### PR TITLE
Issue 41 stopping the machine should ask for confirmation

### PR DESCRIPTION
**Pull request by TomBruyneel**

### CODE REVIEW

File: httpServer/ui/components/dashboard.js [hunk 1]
```diff
@@ -340,6 +345,30 @@ export default class Dashboard extends React.Component {
         });
     }
 
+    toggleActiveState() {
+        this.saveSetting('ACTIVE', this.state.settings.ACTIVE === 0 ? 1 : 0);
+    }
+
+    closeButton_OnClick() {
```
Comment (TomBruyneel) [inline httpServer/ui/components/dashboard.js:352]: not consistent with the rest of the naming in this project. Also it's not really a closebutton, more of an on/off button. Maybe confirmActiveStateChange would be a better option?

File: httpServer/ui/components/dashboard.js [hunk 2]
```diff
@@ -362,9 +391,31 @@ export default class Dashboard extends React.Component {
                             <SaveIcon size="md" />
                         </button>
                         <button className={'btn ' + (this.state.settings.ACTIVE === 0 ? 'inactive' : 'running')}
-                            onClick={() => this.saveSetting('ACTIVE', this.state.settings.ACTIVE === 0 ? 1 : 0)}>
+                            onClick={() => this.closeButton_OnClick()}>
                             <OnOffIcon size="md" />
                         </button>
+                        <Dialog
+                            open={this.state.showCloseDialog}
```
Comment (TomBruyneel) [inline httpServer/ui/components/dashboard.js:398]: The first attribute can be best left on the same line as the tag to stay consistent with the rest of the code

File: httpServer/ui/components/dashboard.js (hunk 1 above)
Comment (ftsolakidis) [reply httpServer/ui/components/dashboard.js:371]: changed to confirmActiveStateChange

File: httpServer/ui/components/dashboard.js (hunk 2 above)
Comment (ftsolakidis) [reply httpServer/ui/components/dashboard.js:417]: Done as requested

**Pull request merged**